README.md

```diff
@@ -36,13 +36,12 @@ By default, the repo links everything under `src` for local development.  Someti
 
 ## Publishing
 You can publish npm modules from this repo by running:
-```lerna publish```
+```lerna publish``` from the main folder.
 Publishing from sub-folders using `npm publish` will result in broken components - don't do this!
-Please only use `custom version` and do the bump manually.
 
-`storybook` repo is private - Its version will be bumped, but it will not be published
+If you have bumped `package.json` as a part of your work, please use `custom version` and release the version that is in your `package.json`.  If you have not yet bumped `package.json` because you made no changes, but lerna asks you to bump your package, please apply a patch bump.  (It is likely that `package-lock.json` was changed.)  
```

Comment:
If you have already bumped `package.json`:
1. Select the `custom version` option
2. Re-type the version you've specified in your `package.json`
3. ALWAYS bump the `storybook` package with a patch.

If you have NOT already bumped `package.json`:
1. Select `patch` option
2. ALWAYS bump the `storybook` package with a patch.

(Bullets are often easier to read in technical docs.)

Reply:
sounds good!